modules/serverless-config-replicas-eventing.adoc

```diff
@@ -0,0 +1,51 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-ha.adoc
+
+[id="serverless-config-replicas-eventing_{context}"]
+= Configuring high availability replicas for Eventing
+
```

Comment:
Would you mind adding a brief introductory paragraph for the new procedure modules, please? This request is based on module guidelines: https://github.com/redhat-documentation/modular-docs/blob/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

Reply:
Sure, done.